sdv/metadata/anonymization.py

```diff
@@ -56,15 +57,15 @@ def is_faker_function(function_name):
         True if the ``function_name`` is know to ``Faker``, otherwise False.
     """
     try:
-        getattr(Faker(), function_name)
+        getattr(Faker(AVAILABLE_LOCALES), function_name)
```

Comment:
What does this do?

Reply:
Since some Faker functions are only available in specific locales, this will let us resolve functions from any of the available locales. Previously, if someone tried to use a function not available in en_US, this would error. Since the metadata doesn't have access to the synthesizer locale we want to use, we just have to check all of them to see if it's a valid function.

Reply:
Can we add a test for such an example and then :shipit: